### ISSUE TITLE
BMC: Tell u-boot where squashfs image is

### ISSUE THREAD

**Issue by anoo1** [openbmc/openbmc#1943]
Set the u-boot env var to tell u-boot where the squashfs image (which ubi vol) and boot from there.

**Comment by anoo1** [comment]
Status:
07/13 - New function in the bmc code update script to set the bootargs variable to the name of the volume where the squashfs image is.

**Comment by anoo1** [comment]
With a kernel with changes from #1935:

```
bootargs=console=ttyS4,115200n8 ubi.mtd=pnor,0,0,6 ubi.block=6,6 ro rootfstype=squashfs root=/dev/ubiblock6_6

# df -T
Filesystem           Type       1K-blocks      Used Available Use% Mounted on
/dev/root            squashfs       14336     14336         0 100% /

# mount
/dev/ubiblock6_6 on / type squashfs (ro,relatime)
```

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/5911> uboot: Add mtd support and env variables to boot kernel and rootfs
Resolves  #1943 BMC: Tell u-boot where squashfs image is
